src/main/java/envoy/event/GroupResizeEvent.java

```diff
@@ -0,0 +1,97 @@
+package envoy.event;
+
+import envoy.data.Contact;
+import envoy.data.Group;
+import envoy.data.User;
+
+/**
+ * This event is used to communicate changes in the group size between client
+ * and server.<br>
+ * Possible actions are adding or removing certain {@link User}s to or from a
+ * certain {@link Group}
+ * <br>
+ * Project: <strong>envoy-common</strong><br>
+ * File: <strong>GroupResizeEvent.java</strong><br>
+ * Created: <strong>25 Mar 2020</strong><br>
+ *
+ * @author Leon Hofmeister
+ * @since Envoy Common v0.1-beta
+ */
+public class GroupResizeEvent extends Event<Long> {
+
+	private final long		groupID;
+	private OperationType	operationType;
+
+	private static final long serialVersionUID = 0L;
+
+	/**
+	 * This enum defines all possibilities for handling
+	 * {@link GroupResizeEvent}s.<br>
+	 * These can be: {@link OperationType#ADD} or {@link OperationType#REMOVE}.<br>
+	 * <br>
+	 * Project: <strong>envoy-common</strong><br>
+	 * File: <strong>GroupResizeEvent.java</strong><br>
+	 * Created: <strong>25 Mar 2020</strong><br>
+	 *
+	 * @author Leon Hofmeister
+	 * @since Envoy Common v0.1-beta
+	 */
+	public enum OperationType {
```

Comment:
Change this to `Operation` to match the class `ContactOperationEvent`.
```suggestion
	public enum Operation {
```